data/stoa0104a/stoa012/stoa0104a.stoa012.opp-lat1.xml

```diff
@@ -40,25 +42,28 @@
             </publicationStmt>
             
             <sourceDesc>
-                <listBibl xml:lang="lat">
+                <listBibl>
                     <biblStruct>
-                        <monogr>
-                            <editor>
-                                <persName>
-                                    <name xml:lang="lat">Guilelmus Hartel</name>
-                                    <addName xml:lang="de">Wilhelm Hartel</addName>
-                                </persName>
-                            </editor> 
-                          <author ref="http://data.perseus.org/catalog/urn:cite:perseus:cattg.1808.1">Cyprianus</author>  
-                            <title>Epistulae</title>
-                            <imprint>
-                                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                                <pubPlace>Wien, Österreich</pubPlace>
-                                <date>1871</date>
-                            </imprint>
-                            <biblScope unit="volume">3.2</biblScope>
-                        </monogr>
-                        <ref target="http://archive.org/stream/corpusscriptoru16wissgoog#page/n635/mode/2up">Internet Archive</ref>
+                      <monogr>
+                        <editor>
+                          <persName>
+                            <name>Wilhelm von Hartel</name>
+                          </persName>
+                        </editor> 
+                        <author>Cyprian, Saint</author>  
+                        <title xml:lang="lat">S. Thasci Caecili Cypriani Opera omnia</title>
+                        <imprint>
+                          <publisher>Gerold</publisher>
+                          <pubPlace>Vienna</pubPlace>
+                          <date>1871</date>
+                        </imprint>
+                        <biblScope unit="volume" xml:lang="lat">Pars II</biblScope>
+                      </monogr>
+                      <series>
+                        <title xml:lang="lat">Corpus Scriptorum Ecclesiasticorum Latinorum</title>
+                        <biblScope unit="volume">3.2</biblScope>              
+                      </series>
+                        <ref target="">Internet Archive</ref>
```

Comment:
FYI The empty ref target is causing the testing failure.

Reply:
You are brilliant! thanks for finding this!